app/templates/suppliers/opportunities_overview.html

```diff
@@ -0,0 +1,63 @@
+{% extends "_base_page.html" %}
+{% import "toolkit/summary-table.html" as summary %}
+{% block page_title %}Opportunities Overview - Digital Marketplace{% endblock %}
+
+{% block breadcrumb %}
+  {%
+    with items = [
+    {
+        "link": "/",
+        "label": "Digital Marketplace"
+    },
+    {
+        "link": url_for(".dashboard"),
+        "label": "Your account"
+    }
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+
+    <div class="grid-row">
+
+        <div class="column-two-thirds">
+            {% with
+                heading = 'Your {} opportunities'.format(framework.name)
+            %}
+            {% include 'toolkit/page-heading.html' %}
+            {% endwith %}
+        </div>
+    </div>
+
+  <p>
+    <a href={{ "/{}/opportunities".format(framework.framework) }}>
+      Search for other opportunities on the Digital Marketplace
+    </a>
+  </p>
+
+{{ summary.heading("Completed", id="completed-opportunities") }}
+{% call(item) summary.list_table(
+    completed|sort(attribute='submittedAt', reverse=True),
+    caption="Completed opportunities",
+    empty_message="You haven’t applied to any opportunities",
+    field_headings=[
+        "Opportunity",
+        "Application deadline",
+        summary.hidden_field_heading("Remove")
+    ],
+    field_headings_visible=True
+) %}
+
+    {% call summary.row() %}
+        {% call summary.field() %}
+            <a href={{ "/{}/opportunities/{}".format(framework.framework, item.briefId) }}>{{ item.brief.title }}</a>
+        {% endcall %}
+        {{ summary.text(item.brief.applicationsClosedAt|dateformat) }}
+        {{ summary.link("View application", url_for(".view_response_result", brief_id=item.briefId)) if not item.brief.status == 'withdrawn' else summary.text() }}
```

Comment:
This from Tobi:
> Putting the text withdrawn is confusing because it’s does not clearly state who withdrew it or what was withdrawn. Since the other link goes to response, suppliers might be think their application was withdrawn

To further the above the page for the opportunity (currently linked to from the brief title) will contain the withdrawn banner. 
In the future there will be a link in the empty section but we need to decide on what to do with the brief response page itself before we link to it because at the moment it has all of the 'What happens next' text on it which won't apply to a withdrawn brief.